scripts/dataset/export/index.js

```diff
@@ -0,0 +1,124 @@
+import fsApi from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import archiver from 'archiver';
+import config from 'config';
+
+import GitAdapter from '../../../src/storage-adapters/git/index.js';
+import MongoAdapter from '../../../src/storage-adapters/mongo/index.js';
+import * as renamer from '../../utils/renamer/index.js';
+import readme from '../assets/README.template.js';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const locale = 'en-EN';
+const dateOptions = { year: 'numeric', month: 'long', day: 'numeric' };
+
+export default async function generateArchive({ archivePath, releaseDate }) {
```

Comment:
Why do we need `releaseDate`? Shouldn't it be obtained locally from the current date?
```suggestion
export default async function generate({ archivePath, releaseDate }) {
```

Reply:
It's to share the exact same date between dataset generation and publication on github. To ensure the message that will be with the release and the readme inside the dataset will have the same date.